src/Compilers/Core/Portable/Emit/Context.cs

```diff
@@ -42,6 +42,19 @@ public EmitContext(CommonPEModuleBuilder module, SyntaxNode syntaxNodeOpt, Diagn
             _flags = flags;
         }
 
+        public EmitContext(
+            CommonPEModuleBuilder module,
+            SyntaxNode? syntaxNodeOpt = null,
+            RebuildData? rebuildDataOpt = null,
+            DiagnosticBag? diagnostics = null,
+            bool metadataOnly = false,
+            bool includePrivateMembers = true)
```

Comment:
Are we relying on any of these default values currently? If not, consider removing the default values since it feels like callers should be explicit about some of these values, particularly `diagnostics`.

Reply:
Took this approach to minimize the diff for now. Wanted to wait on @tmat to give feedback on whether this was an appropriate way to pass around `RebuildData`. If he confirms I'll remove all the default values here and clean up the callers.